plugins/source/heroku/client/spec.go

```diff
@@ -13,6 +13,8 @@ import (
 type Spec struct {
 	Token string `yaml:"token,omitempty" json:"token"`
 
+	// These don't actually work (throw a "json unknown field" error) - but not a high-piority to fix this,
```

Comment:
I think it'll work if we change `yaml` to `json` in all the cases here. But yeah, not a high priority.

Reply:
In fact, maybe we should just remove these options until further notice?

Reply:
Will do in a follow up PR

Reply:
Yup, not a blocker for this PR